pandas/core/reshape/reshape.py

```diff
@@ -852,7 +852,7 @@ def lreshape(data, groups, dropna=True, label=None):
 
 
 def wide_to_long(df, stubnames, i, j, sep="", suffix='\d+'):
-    r"""
+    """
     Wide panel to long format. Less flexible but more user-friendly than melt.
 
```

Comment:
need a node that the casting will occur (with a versionadded tag)

Reply:
I added a one line explanation, followed by the versionadded tag. Not sure if thats correct